src/fielddef.ts

```diff
@@ -169,8 +169,9 @@ export interface ScaleFieldDef<F> extends FieldDef<F> {
 
   /**
    * Sort order for the encoded field.
-   * Supported `sort` values include `"ascending"`, `"descending"`, `"array of field's domains"` and `null` (no sorting).
+   * Supported `sort` values include `"ascending"`, `"descending"`, `"array specifying the preferred order of values"` and `null` (no sorting).
```

Comment:
`"array specifying the preferred order of values"` suggests that this is an enum array, so it's a bit misleading. 

`"ascending"`, `"descending"`, `null`, or an array specifying the preferred order of values